LICENSE

```diff
@@ -1,11 +1,7 @@
-MIT License (MIT)
+MIT License
 
-<!-- lint disable no-auto-link-without-protocol -->
-
-Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com> (http://arcticicestudio.com)  
-Copyright (C) 2018-present Sven Greb <code@svengreb.de> (http://svengreb.de)
-
-<!-- lint enable no-auto-link-without-protocol -->
+Copyright (c) 2018-present Arctic Ice Studio <development@arcticicestudio.com> (http://arcticicestudio.com)
+Copyright (c) 2018-present Sven Greb <code@svengreb.de> (http://svengreb.de)
```

Comment:
When we're at it, then this can be fixed too 😄 

```suggestion
Copyright (c) 2018-present Arctic Ice Studio <development@arcticicestudio.com> (https://www.arcticicestudio.com)
Copyright (c) 2018-present Sven Greb <development@svengreb.de> (https://www.svengreb.de)
```

Reply:
Are your sites down right now? I get "Page Not Found" when I visit both of those links.

Reply:
I pushed the changes you requested.

Reply:
Thanks!
Someone might think, but in fact they've never been up 😄 
Someone also already noticed this and I've posted a more detailed answer in https://github.com/arcticicestudio/nord-docs/issues/193